src/Screens/EntryScreens/NumberSignUp.js

```diff
@@ -76,6 +76,27 @@ const NumberSignUp = () => {
             <PhoneInput
               value={phoneNumber}
               defaultCountry="IN"
+              phoneInputStyles={{
+                container: {borderWidth: 0, backgroundColor: 'transparent'},
+                input: {
+                  width: '70%',
+                  height: 45,
+                  backgroundColor: '#fff',
+                  margin: 5,
```

Comment:
Remove inline styles and move them to a separate stylesheet

Reply:
Rectified